rest-api/dao/participant_summary_dao.py

```diff
@@ -315,6 +315,9 @@ def _add_order_by(self, query, order_by, field_names, fields):
 
   def make_query_filter(self, field_name, value):
     """Handle HPO and code values when parsing filter values."""
+    from model.config_utils import from_client_biobank_id
```

Comment:
Is this a datastore dependency? Is that why you aren't importing normally? If so, please just add a comment here.

Reply:
Nope, I was experimenting and forgot to move it. It's moved now 😄 
Also, shouldn't our linters catch that ?